app/renderer/js/main.ts

```diff
@@ -400,8 +400,16 @@ class ServerManagerView {
 
 	initLeftSidebarEvents(): void {
 		this.$dndButton.addEventListener('click', () => {
-			const dndUtil = DNDUtil.toggle();
-			ipcRenderer.send('forward-message', 'toggle-dnd', dndUtil.dnd, dndUtil.newSettings);
+			const dnd = ConfigUtil.getConfigItem('dnd', false);
+			if (dnd) {
+				const dndUtil = DNDUtil.toggle();
+				ipcRenderer.send('forward-message', 'toggle-dnd', dndUtil.dnd, dndUtil.newSettings);
+			}
+			if (document.querySelector('.dnd-menu') === null) {
```

Comment:
Should be an `else if` block.

Reply:
@manavmehta can you make the above changes?